modules/ip/datapath/ip_fragment.c

```diff
@@ -34,7 +40,7 @@ enum {
 
 static uint16_t
 ip_fragment_process(struct rte_graph *graph, struct rte_node *node, void **objs, uint16_t nb_objs) {
-	struct rte_mbuf *mbuf, *frag_mbuf;
+	struct rte_mbuf *mbuf, *frag_mbuf, *frags[MAX_FRAGMENTS];
```

Comment:
Since the node enqueue helper expects a void * array, and this frags[] array is not used for anything else, I would declare it as void *frags[].
This avoids the cast later.